client/dashboard/sites/settings-site-visibility/privacy-form.tsx

```diff
@@ -61,7 +64,17 @@ export function PrivacyForm( {
 
 	const handleSubmit = ( e: React.FormEvent ) => {
 		e.preventDefault();
-		mutation.mutate( { ...formData } );
+		mutation.mutate(
+			{ ...formData },
+			{
+				onSuccess: () => {
+					createSuccessNotice( __( 'Settings saved.' ), { type: 'snackbar' } );
```

Comment:
We will likely need this code in most of the setting screens, so we should look into avoiding its duplication.

Reply:
There's also DOTCOM-13147 which looks like it would live in a similar shared place.

Reply:
Good point, it'd be nice if we can have this polish without explicitly re-implementing them for each form.